samples/operate-first/Makefile

```diff
@@ -0,0 +1,11 @@
+export RELEASE_NAME=m4d
+export NAMESPACE=m4d-system
+export M4D_CHART_PATH=../../charts/m4d
```

Comment:
Add the following to the top of the Makefile
```
ROOT_DIR := ../..
include $(ROOT_DIR)/Makefile.env
```

Then use `$(ROOT_DIR)` in this line and the next one